Assessment/About.html

```diff
@@ -9,7 +9,7 @@
 <body>
     <div id="WholePage">
         <div id="Logo">
-            <img id="LogoImg" src = "JadieLavender_Logo.png" alt = "Jadie Lavender Logo.">
+            <img id="LogoImg" src = "JadieLavender_Logo.png" alt= "Jadie Lavender Logo.">
```

Comment:
Remove spaces between src/alt, = and the quoted text, so src = "JadieLavender_Logo.png" should be src="JadieLavender_Logo.png"
Although those spaces would probably pass a html compliance checker, they could cause screen-readers to act unexpectedly.